vcs/vcs_gogit.go

```diff
@@ -81,50 +78,6 @@ func (g *GoGitVCS) assertValidRepo() error {
 	return nil
 }
 
-func (g *GoGitVCS) goGitInitAndCommitAll(ctx context.Context, msg string) error {
-	if err := ensureFolderExists(g.Path); err != nil {
-		return err
-	}
-	if g.isRepoOpen() {
-		return errors.New("Repo already exists. Repo: " + g.Path)
-	}
-	fs := osfs.New(g.Path)
-	dot, _ := fs.Chroot(".git")
-	cache := cache.ObjectLRU{
-		MaxSize: 100 * cache.MiByte,
-	}
-	storage := filesystem.NewStorage(dot, &cache)
-	if storage == nil {
-		return errors.New("Unable to create new storage")
-	}
-	r, err := git.Init(storage, fs)
-	if err != nil {
-		err = gz.WithStack(err)
-		gz.LoggerFromContext(ctx).Info("Error while doing git Init. Err: " + fmt.Sprint(err) + ". Repo: " + g.Path)
-		return err
-	}
-	g.r = r
-
-	// Add all files to git index
-	w, _ := r.Worktree()
-	if _, err := w.Add("."); err != nil {
-		return gz.WithStack(err)
-	}
-
-	// Commit
-	_, err = w.Commit(msg, &git.CommitOptions{
-		Author: &object.Signature{
-			Name:  gitName,
-			Email: gitEmail,
-			When:  time.Now(),
-		},
-	})
-	if err != nil {
-		return gz.WithStack(err)
-	}
-	return nil
-}
```

Comment:
Why was this removed?

Reply:
This function was not being used, the linter recommends removing it.

Reply:
Thanks!